.github/workflows/mvn-verify-check.yml

```
@@ -30,7 +30,45 @@ env:
     -Dskip
 
 jobs:
+  cache-dependencies:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [ '8', '11', '17' ]
```

The external dependencies are not JDK-sensitive currently,  the cache key does not include the jdk version,
doing only 8 is sufficient

```suggestion
        java: [ '8' ]
```